create_config.yml

```diff
@@ -6,4 +6,8 @@
   tasks:
 
     - import_tasks: tasks/set_vars.yml
-    - import_tasks: tasks/create_config.yml
+    - include_tasks: tasks/create_app_config.yml
```

Comment:
Why don't use tasks/run_tasks_for_apps.yml helper ?

Reply:
You are right: it's not consistent. Let's remove this unused playbook instead.